source/openapi-admin-v3.yaml

```
@@ -4387,6 +4387,137 @@ paths:
         "204":
           description: No Content
 
+  "/groups/{groupId}/apps/{appId}/edge":
+    parameters:
+      - "$ref": "#/components/parameters/GroupId"
+      - "$ref": "#/components/parameters/AppId"
+
+    get:
+      tags:
+        - edge-server
+      operationId: adminGetAllEdgeServerInstances
+      summary: List Edge Servers
+      description: |-
+        List all [Edge Server](https://www.mongodb.com/docs/atlas/app-services/edge-server/) 
+        instances for the given App.
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  edge_servers:
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/EdgeServerInstance"
+
+    post:
+      tags:
+        - edge-server
+      operationId: adminCreateEdgeServerInstance
+      summary: Create an Edge Server instance
+      description: Create a new Edge Server instance.
```

Do we want to mention any of the registration process here? e.g.
```suggestion
      description: Create a new Edge Server instance. The response returns a registration key (token?) that you use to authenticate Edge Server on the host.
```